PowerEditor/installer/nativeLang/english.xml

```diff
@@ -273,6 +273,23 @@
                     <Item id="10003" name="Move to New Instance"/>
                     <Item id="10004" name="Open in New Instance"/>
 
+                    <Item id="45060" name="Big5 (Traditional)"/>
+                    <Item id="45061" name="GB2312 (Simplified)"/>
+                    <Item id="45054" name="OEM 861 : Icelandic"/>
+                    <Item id="45057" name="OEM 865 : Nordic"/>
+                    <Item id="45053" name="OEM 860 : Portuguese"/>
+                    <Item id="45056" name="OEM 863 : French"/>
+
+                    <Item id="46033" name="Assembly"/>
+                    <Item id="46022" name="Batch"/>
+                    <Item id="46026" name="Fortran (free form)"/>
+                    <Item id="46019" name="INI file"/>
+                    <Item id="46015" name="MS-DOS Style"/>
+                    <Item id="46016" name="Normal Text"/>
+                    <Item id="46017" name="Resource file"/>
+                    <Item id="46027" name="Shell"/>
+                    <Item id="46058" name="Fortran (fixed form)"/>
+
```

Comment:
Could you tell me the reason why these are "invalidated"? It seems like these are still present in notepad++ 7.1 and translating them in japanese.xml changes texts of the menu.